### PR TITLE
Remove usage of TaskDone.Done, which is obsolete

### PR DESCRIPTION
**Pull request by xiazen**
Remove usage of obsolete use of TaskDone.Done, which was introduced in couple new PRs. 

@ReubenBond , you shouldn't use it anymore either ! :D

### CODE REVIEW

File: src/OrleansServiceBus/Providers/EventDataGeneratorStreamProvider/EeventHubPartitionGeneratorReceiver.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ public void ConfigureDataGeneratorForStream(IStreamIdentity streamId)
 
         public Task CloseAsync()
         {
-            return TaskDone.Done;
+            return Task.CompletedTask;
```
Comment (ReubenBond) [inline src/OrleansServiceBus/Providers/EventDataGeneratorStreamProvider/EeventHubPartitionGeneratorReceiver.cs]: not related to this change, but this file's name is a typo, I think: EeventHubXXX

**Comment by ReubenBond** [comment]
@dotnet-bot test netfx-functional

File: src/OrleansServiceBus/Providers/EventDataGeneratorStreamProvider/EeventHubPartitionGeneratorReceiver.cs (hunk 1 above)
Comment (xiazen) [inline src/OrleansServiceBus/Providers/EventDataGeneratorStreamProvider/EeventHubPartitionGeneratorReceiver.cs]: let me fix that before this !!

**Comment by ReubenBond** [comment]
@dotnet-bot test netfx-functional

**Comment by jdom** [comment]
@dotnet-bot test netfx-functional

**Comment by jdom** [comment]
@dotnet-bot test netfx-functional

**Comment by jdom** [comment]
@dotnet-bot test netfx-functional

**Pull request merged**